index.html

```diff
@@ -1,61 +1,117 @@
             no matter what your background might be.
         </p>
 
-        <p> <strong>24/7 UNLIMITED ACCESS FOR FREE!</strong>
-            
+        <p class="introt"> <strong>24/7 UNLIMITED ACCESS FOR FREE!</strong>
+
         </p>
-        <p>Get your mobile devices ready and let rock 'n' roll!!!</p>
-        
+        <p class="introt">Get your mobile devices ready and let's rock 'n' roll!!!</p>
+
     </section>
 
     <section class="program-list">
         <h2>Main Program</h2>
-        
-        <ul class="items">
+        <hr>
+        <ul class="program-items">
+            <li>
+                <img src="assets/images/html.png" alt="#" class="html">
+                <h4>HTML5</h4>
+                <p>a markup language used for structuring and presenting content on the World Wide Web.</p>
+            </li>
+            <li><img src="assets/images/css.png" alt="#" class="html">
+                <h4>CSS3</h4>
+                <p>Used in the styling and formatting of Web pages. </p>
+            </li>
             <li>
-                <img src="" alt="" class="html">
-                <p>HTML5</p>
-                <p>The skeleton of webpages</p>
+                <img src="assets/images/javascript.png" alt="#" class="html">
+                <h4>Javascript</h4>
+                <p>97% of websites use JavaScript on the client side for web page behavior, 
+                    often incorporating third-party libraries.</p>
             </li>
-            <li><img src="" alt="" class="html">
-                <p>CSS3</p>
-                <p>The skeleton of webpages</p></li>
             <li>
-                <img src="" alt="" class="html">
-                <p>Javascript</p>
-                <p>The skeleton of webpages</p>
+                <img src="assets/images/bootstrap.png" alt="#" class="html">
+                <h4>Frameworks</h4>
+                <p>A framework is a structure that you can build software on. It serves as a foundation,
+                     so you're not starting entirely from scratch.</p>
+            </li>
+            <li><img src="assets/images/python.png" alt="#" class="html">
+                <h4>Backend Tech</h4>
+                <p> place that contains all the data and relevant information that is to be shown to the visitors with the help of a browser</p>
             </li>
-            <li> <img src="" alt="" class="html">
-                <p>Frameworks</p>
-                <p>The skeleton of webpages</p></li>
-            <li> <img src="" alt="" class="html">
-                <p>Backend Technologies</p>
-                <p>The skeleton of webpages</p></li>
         </ul>
 
         <button>
-            Say hello to a dimension
+            Say hello to a new dimension
         </button>
     </section>
 
-    <section class="tutors">
-        <h3> Our Trainers</h3>
+    <section class="trainers">
     </section>
+
+    <footer class="home-footer">
```

Comment:
- [ ] Kindly take a look at your footer on the mobile version, because it's not being shown there. Here's an image to demonstrate:
![Screenshot_2022-02-09_20-05-14](https://user-images.githubusercontent.com/59100477/153271985-62f853dd-e344-425c-bd49-f3dbbaa39428.png)
I have to say that I'm at the bottom of the page and the footer is not there.

- [ ] Related to the footer, also notice that for the desktop version, your footer is not properly aligned. The logo is on the left side but has some margin applied to the left, but the text which says `Some rights reserved` it's on the right side with no margin or padding, which make the design not look as intended, so kindly take a look and fix this issue.

Reply:
fixed it